hexrd/config/findorientations.py

```diff
@@ -23,6 +24,44 @@
 
 class FindOrientationsConfig(Config):
 
+    _find_ori = "find-orientations"
+
+    def _active_material_str(self):
+        return self.parent.material.active.strip().replace(' ', '-')
```

Comment:
My thinking here was that a property is part of the external API for the module. Since I only plan to use this internally, then making it property might be confusing.

Reply:
Okay that sounds fine!